src/index.js

```diff
@@ -0,0 +1,72 @@
+import _, { indexOf } from 'lodash';
+import './index.css';
+import {TaskList} from './TasklistFile.js';
+import {Interaction} from './interactive_List.js'
+const textInputAdd = document.getElementById('addTT');
+const taskList = document.getElementById('task-list');
+function check(status) {
+  if (status === true) { return "checked";
+ } else { return "unchecked";
+ }}
+function styleTask(status) { if (status === true) { 
+  return "task taskStyle" } else { return "task" 
+  }  }
+function display() {
+  TaskList.taskArray = JSON.parse(localStorage.getItem('coward')) || [];
+  taskList.innerHTML = ''
+ TaskList.taskArray.forEach((task, i) => {
```

Comment:
In this line, you declared a parameter `task`, that is never used afterwards, so please remove this parameter.